lib/include/openamp/rpmsg_virtio.h

```diff
@@ -75,8 +75,26 @@ struct rpmsg_virtio_device {
 	struct rpmsg_virtio_shm_pool *shpool;
 };
 
-#define RPMSG_REMOTE	VIRTIO_DEV_SLAVE
-#define RPMSG_MASTER	VIRTIO_DEV_MASTER
+#define RPMSG_REMOTE	VIRTIO_DEV_DEVICE
+#define RPMSG_HOST	VIRTIO_DEV_DRIVER
```

Comment:
hare you sure that `VIRTIO_DEV_DRIVER` and 'VIRTIO_DEV_DEVICE'  are defined at this step?
else should be put after the pragma messgaes
```
#ifdef VIRTIO_DEV_MASTER 
[...]
+ #ifndef VIRTIO_DEV_DRIVER
+ #define VIRTIO_DEV_DRIVER
+ #endif
#endif

#ifdef VIRTIO_DEV_SLAVE
[...]
+ #ifndef VIRTIO_DEV_DEVICE
+ #define VIRTIO_DEV_DEVICE
+ #endif
#endif

#define RPMSG_REMOTE	VIRTIO_DEV_DEVICE
#define RPMSG_HOST	VIRTIO_DEV_DRIVER
```

Reply:
They are defined in openamp/virtio.h, which is included at the top of the file, so this should be fine. I'm not sure we need the ifdef checks and warnings for the old names, since the only place these constants are ever defined is in the header file.